ADR/0023-release-and-releaseplan-authorization.md

```diff
@@ -0,0 +1,55 @@
+# Release and ReleasePlan Authorization
+
+Date: 2023-04-17
+
+## Status
+
+Accepted
+
+## Context
+
+* It is imperative to know which associate has authorized a change to production in order to have accountability for changes.
+* Flexibility is required when specifying authorization either on a per-release basis or as a standing authorization.
+
+## Decision
+
+### Architecture Overview
+#### Terminology
+
+- `Release`: A CRD that describes what should be released and using which plan.
+- `ReleasePlan`: A CRD that describes where and how an application should be released.
+- `ReleaseStrategy`: A CRD that describes which pipeline to execute along with specific pipeline parameters.
+
+#### Architecture
+
+![](../diagrams/ADR-0023/ReleaseAndReleasePlanAuthorization.jpg)
+
+##### Capturing authorization data
+* The idea is that when a release pipeline executes, the `Pipeline`'s `Tasks` have access to the name of the associate that authorized the change in production.
```

Comment:
What is this idea? Are you trying to defer how the release pipeline has access to the authorized associate as an implementation detail? If so, I think we can make that clearer, maybe as a precondition before the bullets start.